cli/cmd/uninstall.go

```diff
@@ -61,6 +61,10 @@ var uninstallCmd = &cobra.Command{
 			}
 		}
 		logging.PrintLog("Successfully uninstalled keptn", logging.InfoLevel)
+		logging.PrintLog("Note: Some namespaces and services have not been uninstalled. Please inspect this yourself using", logging.InfoLevel)
+		logging.PrintLog("      kubectl get pods --all-namespaces", logging.InfoLevel)
```

Comment:
Be more specific?
"You can delete those resources by 
`kubectl delete namespace istio-system`
`kubectl delete all -l app=helm -n kube-system`
`kubectl delete namespace dynatrace`

Reply:
Again, being less specific here, as at some point it might not be istio, but maybe linkerd, and it might not be helm but kustomize.

Reply:
Same argument as above.

Reply:
I've rewritten the code, see above discussion (I'll close this one)